frontend/src/components/form/base/BaseComponent.vue

```diff
@@ -16,8 +16,10 @@
       v-on="$listeners"
     >
       <!-- passing through all slots -->
-      <slot v-for="(_, name) in $slots" :slot="name" :name="name" />
-      <template v-for="(_, name) in $scopedSlots" :slot="name" slot-scope="slotData">
+      <template #[name]>
+        <slot v-for="(_, name) in $slots" :name="name" />
+      </template>
+      <template v-for="(_, name) in $slots" #[name]="slotData">
```

Comment:
Is this really the same as before?

Reply:
no, its what the linter rule proposes, which is :hankey: 

Now i tried another way, which also did not yet work.

With the new syntax, it seems difficult to pass all slots all all slots with slotData.

Reply:
@BacLuc I think I still don't fully understand the scoping. What is the difference between the `<slot v-for>` and `<template v-for="$scopedSlots">` ?

Reply:
Maybe this will only work in the future with vue3?

Reply:
Me neither.
That's why i now put a step in between with https://github.com/ecamp/ecamp3/pull/5170
where i do the other parts i understood.